File: Desafio - C#/Verificar_Url.cs
using System;

class Program
{
    static void Main()
    {
        Console.WriteLine("Digite a URL do site em questão: ");
        string url = Console.ReadLine();

        string resultado = VerificarUrl(url);

        // Exibe o resultado
        Console.WriteLine(resultado);
    }

    static string VerificarUrl(string url)
    {
        if (url.StartsWith("https://"))
        {
            return "URL segura";
        }
        else if (url.StartsWith("http://"))
        {
            return "URL não segura";
        }
        else
        {
            return "Formato inválido";
        }
    }
}

File: Desafio - KOTLIN/Verificar_Url.kt
fun verificarUrl(url: String): String {
    return when {
        url.startsWith("https://") -> "URL segura"
        url.startsWith("http://") -> "URL não segura"
        else -> "Formato inválido"
    }
}

fun main() {
    print("Digite a URL do site em questão: ")
    val url = readLine().orEmpty()  // Usa orEmpty() para garantir que a string não seja nula
    val resultado = verificarUrl(url)

    println(resultado)
}
